### PR TITLE
Control plane status

### PR DESCRIPTION
**Pull request by brancz**
Adding the Kubernetes control plane status dashboard back.

@metalmatze

### CODE REVIEW

File: example.jsonnet [hunk 1]
```diff
@@ -1,12 +1,16 @@
 local k = import "ksonnet.beta.3/k.libsonnet";
+local service = k.core.v1.service;
+local servicePort = k.core.v1.service.mixin.spec.portsType;
 local grafana = import "grafana/grafana.libsonnet";
-local namespace = "monitoring";
+local namespace = "monitoring-grafana";
```
Comment (metalmatze) [inline example.jsonnet]: Why did you change this namespace? I know you're supposed to copy this file, but asking anyway.

File: example.jsonnet [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
 k.core.v1.list.new([
     grafana.dashboardDefinitions.new(namespace),
     grafana.dashboardSources.new(namespace),
     grafana.dashboardDatasources.new(namespace),
     grafana.deployment.new(namespace),
     grafana.serviceAccount.new(namespace),
-    grafana.service.new(namespace),
+    grafana.service.new(namespace) +
+        service.mixin.spec.withPorts(servicePort.newNamed("http", 3000, "http") + servicePort.withNodePort(30902)) +
```
Comment (metalmatze) [inline example.jsonnet]: Can't we move those mixins into the service file? Or is this an example of overriding the service?

File: src/kubernetes-jsonnet/grafana/configs/dashboard-definitions/kubernetes-control-plane-status-dashboard.libsonnet [hunk 3]
```diff
@@ -0,0 +1,63 @@
+local grafana = import "grafonnet/grafana.libsonnet";
+local dashboard = grafana.dashboard;
+local prometheus = grafana.prometheus;
+local row = grafana.row;
+local graphPanel = grafana.graphPanel;
+
+local promgrafonnet = import "promgrafonnet/promgrafonnet.libsonnet";
+local gauge = promgrafonnet.gauge;
+
+local apiservers_up          = gauge.new("API Servers UP",         '(sum(up{job="apiserver"} == 1) / sum(up{job="apiserver"})) * 100');
```
Comment (metalmatze) [inline src/kubernetes-jsonnet/grafana/configs/dashboard-definitions/kubernetes-control-plane-status-dashboard.libsonnet]: Why not put the queries on new lines? :thinking:

File: example.jsonnet (hunk 2 above)
Comment (brancz) [reply example.jsonnet]: 1) It's an example of overriding
2) I think the sensible default for a package is not to expose a NodePort, people are surely going to get bitten by it if we make that the default of the package.
